camelot/data/types/__init__.py

```diff
@@ -134,11 +134,20 @@ def tooltip(self):
         if self.example is not None:
             return 'e.g: {}'.format(self.example)
 
+    @property
+    def compact_repl(self):
```

Comment:
Dit is nu gedupliceerd met de identifier type, maar dat wordt weggewerkt als de zip code types uiteindelijk ook identifier types worden.